03-task-manager/starter/controllers/tasks.js

```diff
@@ -0,0 +1,27 @@
+const Task = require('../models/tasks')
+
+
+const getAllTasks = (req, res) => {
+  res.send('get all tasks')
+}
+const createTasks = async (req, res) => {
```

Comment:
```suggestion
const createTask = async (req, res) => {
```
I'd probably make this singular, as the purpose is to create one task rather than multiple tasks.